src/tests/ftest/util/mdtest_test_base.py

```diff
@@ -99,7 +108,9 @@ def execute_mdtest(self, out_queue=None, display_space=True):
             self.container.skip_cleanup()
             # Need to set self.container to None to force a creation of a new container
             self.container = None
-        self.stop_dfuse()
+
+        if self.dfuse is not None:
+            stop_dfuse(self, self.dfuse)
```

Comment:
Another case where we should actually call `dfuse.stop()` so we don't ignore errors